packages/core/packages/PackageManager.ts

```diff
@@ -123,18 +123,18 @@ export class PackageManager {
 	}
 
 	public static removePackage(packageName: string, verbose: boolean = false): boolean {
-		let command: string;
 		const managerCommand = this.getManager();
+		let args: string[];
 		switch (managerCommand) {
 			case "npm":
 			/* passes through */
 			default:
-				command = `${managerCommand} uninstall ${packageName} --quiet --save`;
+				args = ['uninstall', packageName, '--quiet', '--save'];
 				break;
 		}
 		try {
 			// tslint:disable-next-line:object-literal-sort-keys
-			Util.execSync(command, { stdio: "pipe", encoding: "utf8" });
+			spawnSync(managerCommand, args, { stdio: "pipe", encoding: "utf8" });
```

Comment:
I wonder if its worth adding spawnSync to Util, and wrap it in a try-catch, like it is with execSync?